scripts/notifyAdmin.cfg

```diff
@@ -0,0 +1,15 @@
+/*  This configuration file is a place for your private information used to send mail.  It is expected
+to be located at /Users/Shared/notifyAdmin.cfg.  Replace the address and password in the text below to use.
+The subject, greeting, and body can also be altered to suit your taste.
+*/
+MailService = "gmail"
+ReturnAddress = "YourAddressHere"
+MailPassword = "YourPasswordHere"
+MailSubject = "Thanks for becoming an admin in the Monthly Calling Campaign"
+MailGreeting = "Welcome, "
+MailBody = ",
```

Comment:
current MailBody:

Your admin account has been approved, and you can now log in at admin.cclcalls.org.  Your username is "_______." You have access to ____ <<for ex. "TX-25" or "TX-1, TX-2, and TX-Sr. Senator">>. If you need access to any other districts, just let me know.

Once you logged in, please:
Review the [Starting a Monthly Calling Campaign training](https://community.citizensclimate.org/resources/item/19/390)
Join the [CCL Community Group](https://community.citizensclimate.org/groups/home/2332) for MCC admins
If you have any questions, you can post into the Community group or email feedback@cclcalls.org.

Thanks,
Ben Boral